analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -289,8 +293,12 @@ private fun parseAuthors(spec: PackageSpec?): SortedSet<String> =
 private fun resolveLocalSpec(definitionFile: File): File? =
     definitionFile.parentFile?.resolve(".nuspec")?.takeIf { it.isFile }
 
-private fun getIdentifier(name: String, version: String) =
-    Identifier(type = "NuGet", namespace = "", name = name, version = version)
+internal fun getIdentifier(name: String, version: String): Identifier {
+    val namespace = name.split('.', limit = 3).toMutableList()
```

Comment:
Maybe the splitting here should be made more strict and also require PascalCase as described in the linked article.